components/logging/org.wso2.carbon.logging.service/src/main/java/org/wso2/carbon/logging/service/data/LogEvent.java

```diff
@@ -114,6 +117,29 @@ public String getMessage() {
         return message;
     }
 
+    /**
+     * Returns the message string for the log event.
+     * If the message string length is greater that {@value MAX_LOG_MESSAGE_LENGTH} then the message string
+     * will be truncated to the length of {@value MAX_LOG_MESSAGE_LENGTH} and returned.
+     * The truncation of the message is handled in server side to avoid client downloading the full log message
+     *
+     * @return message string
+     */
+    public String getTruncatedMessage() {
+        return message.length() > MAX_LOG_MESSAGE_LENGTH
+               ? message.substring(0, MAX_LOG_MESSAGE_LENGTH) + "..."
+               : message;
```

Comment:
Let's move ? mark to first line and keep the two conditional values at the same line, WDYT?